src/include/hip/rpp_hip_common.hpp

```diff
@@ -55,12 +55,14 @@ typedef union { float f1[5];
 typedef union { float f1[6];    float2 f2[3];                                                   }   d_float6;
 typedef union { float f1[7];                                                                    }   d_float7;
 typedef union { float f1[8];    float2 f2[4];   float4 f4[2];                                   }   d_float8;
-typedef union { float f1[9];                                                                    }   d_float9;
+typedef union { float f1[9];    float3 f3[3];                                                   }   d_float9;
 typedef union { float f1[12];   float4 f4[3];                                                   }   d_float12;
+typedef union { float f1[14];                                                                   }   d_float14;
```

Comment:
Rajy has suggested in old gaussian filter PR to not add any new custom datatype anymore other than what 
is existing currently
@r-abishek Please share your thoughts here